frontend/public/components/deployment-config.tsx

```diff
@@ -132,7 +133,16 @@ export const DeploymentConfigsDetails: React.FC<{obj: K8sResourceKind}> = ({obj:
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Config Overview" />
       {dc.spec.paused && <WorkloadPausedAlert obj={dc} model={DeploymentConfigModel} />}
-      <DeploymentPodCounts resource={dc} resourceKind={DeploymentConfigModel} />
+      <PodRingDataContoller
+        namespace={dc.metadata.namespace}
+        knative={false}
```

Comment:
Why is knative false? This will be inconsistent with topology for knative resources. Need to show consistent visualizations.